src/ompl/geometric/planners/rrt/RRTstar.h

```diff
@@ -173,17 +190,57 @@ namespace ompl
                 return pruneStatesThreshold_;
             }
 
+            /** \brief Use a k-nearest search for rewiring instead of a r-disc search. */
+            void setKNearest(bool useKNearest)
+            {
+                useKNearest_ = useKNearest;
+            }
+
+            /** \brief Get the state of using a k-nearest search for rewiring. */
+            bool getKNearest() const
+            {
+                return useKNearest_;
+            }
+
```

Comment:
To my untrained eye, `prm/ConnectionStrategy.h` appears fairly tailored to PRM? There is also the problem that for the r-disc variants, the RRT\* bound is different than the PRM\* bound...